Update @media/resolution index.md

scheinercc opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

<!-- ✍️ Summarize your changes in one or two sentences -->
Add examples for additional resolution unit, dppx, x and dpcm, similar to #31902

### Motivation

<!-- ❓ Why are you making these changes and how do they help readers? -->
In #30393 we ended up changing https://developer.mozilla.org/en-US/docs/Web/CSS/resolution#syntax, but the original request was for https://developer.mozilla.org/en-US/docs/Web/CSS/@media/resolution, which this pull request now includes.

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

scheinercc committed:
Update @media/resolution index.md

Add examples for additional resolution unit, dppx, x and dpcm, similar to #31902

estelle committed:
Merge branch 'main' into patch-3

estelle commented:
Instead of adding new values on there own, let's change two of the three existing dpi values to different resolution units.

scheinercc committed:
Merge branch 'main' into patch-3

scheinercc committed:
Merging examples

Removed new values and changed two of the three existing dpi values to different resolution units instead.

scheinercc commented:
@estelle I pushed a new commit. The wording might need to be rephrased :)

estelle commented on files/en-us/web/css/@media/resolution/index.md:
```diff
@@ -24,22 +24,28 @@ The `resolution` feature is specified as a {{cssxref("&lt;resolution&gt;")}} val
 
-/* Minimum resolution */
-@media (min-resolution: 72dpi) {
+/* Minimum resolution unit `dppx`, `x` */
```
```suggestion
/* Minimum resolution synonym units: `dppx` and `x` */
```

estelle committed:
Update files/en-us/web/css/@media/resolution/index.md

estelle reviewed (approved):
Thank you! 🎉
